src/main/java/io/ebeaninternal/server/persist/Binder.java

```diff
@@ -228,14 +226,8 @@ public Object bindObject(DataBind dataBind, Object value) throws SQLException {
       return values;
 
     } else {
-      ScalarType<?> type = getScalarType(value.getClass());
-      if (!type.isJdbcNative()) {
-        // convert to a JDBC native type
-        value = type.toJdbcType(value);
-      }
-
-      int dbType = type.getJdbcType();
-      bindObject(dataBind, value, dbType);
+      ScalarType type = getScalarType(value.getClass());
+      type.bind(dataBind, value);
```

Comment:
@rbygrave take a look at this change. IMHO we should delegate the bind to the ScalarType.
I fall over this, becase bindObject did not support files. (I added a workaround anyway)